src/app/pages/public/LifeCreatePage/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Button, DatePicker, Form, Input, message } from 'antd';
+import { useTranslation } from 'react-i18next';
+import { useCreateLifeMutation } from '../../../api';
+import BasicLayout from '../../../layouts/BasicLayout';
+
+const LifeCreate = () => {
+    const { t } = useTranslation(['life']);
+
+    const [createLife] = useCreateLifeMutation();
+
+    const onFinish = async values => {
+        const formData = {
+            ...values,
+        };
+
+        formData.birthday = new Date(values.birthday);
+        formData.hobbies = values.hobbies.split(' ');
```

Comment:
try to do that directly in the object definition

Reply:
Could you please explain me a little more? I was a bit unclear about that.